src/config/localfile-config.c

```diff
@@ -447,6 +451,22 @@ int Read_Localfile(XML_NODE node, void *d1, __attribute__((unused)) void *d2)
                 return (OS_INVALID);
             }
 
+        } else if (strcasecmp(node[i]->element, xml_localfile_ignore) == 0) {
+            w_calloc_expression_t(&logf[pl].regex_ignore, EXP_TYPE_PCRE2);
```

Comment:
Regarding the abstraction of the regex type thanks to the `expression` library already designed. It could be very easy and interesting to allow the possibility of choosing the regex type with an attribute. For example:
```xml
<ignore type="osregex">expression</ignore>
```
in the same way that it is currently working for the ruleset (https://github.com/wazuh/wazuh/pull/6480). PCRE2 is still the default one.